packages/nc-gui/components/smartsheet/column/EditOrAdd.vue

```diff
@@ -78,6 +78,18 @@ const uiTypesOptions = computed<typeof uiTypes>(() => {
   ]
 })
 
+const renderColumnType = (columnType: string) => {
+  if (locale.value === 'en') return columnType
```

Comment:
I think this line is not required. If it doesn't match, it will return english one by default.